src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc"),
+    url("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
```

Comment:
You should add separate font-face for each font-weight (one for normal, one for bold)

Reply:
I fixed the problem, please check.